public/components/ClearCacheModal/ClearCacheModal.tsx

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
There is some business logic and API call in this component, please move this Modal to containers directory.